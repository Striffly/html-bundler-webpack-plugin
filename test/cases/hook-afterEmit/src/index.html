<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <link href="./style.css" rel="stylesheet" />
  <script src="./main.js" defer="defer"></script>
</head>
<body>
  <h1>Hello World!</h1>
  <img src="@fixtures/images/apple.png" alt="picture">
  <div class="bg-style">.bg-style</div>
  <div class="app1">app1.css - imported in JS</div>
  <div class="app2">app2.css - imported in JS</div>
  <script src="./app.js"></script>
</body>
</html>