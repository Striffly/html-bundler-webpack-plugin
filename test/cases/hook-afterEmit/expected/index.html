<!DOCTYPE html>
<html>
<head>
  <title>Test</title>
  <link href="style.5f6a8236.css" rel="stylesheet" integrity="sha384-L4ZxY6EorKIpuov40WP0Bp6Mrls5TyMrOWsy4oXOp9i/FGeqnyJU+PWwIzWn3T3z" crossorigin="anonymous">
  <link href="app.5b01e96e.chunk.css" rel="stylesheet" integrity="sha384-kIzKP4FS1ZRJP0kja2MyyxIn2yEXD6AGd9bLgi5ftWXzbrKpzzTRX3lAgebd7swV" crossorigin="anonymous">
<script src="main.5317c1f6.js" defer="defer" integrity="sha384-E4IoJ3Xutt/6tUVDjvtPwDTTlCfU5oG199UoqWShFCNx6mb4tdpcPLu7sLzNc8Pe" crossorigin="anonymous"></script>
</head>
<body>
  <h1>Hello World!</h1>
  <img src="img/apple.02a7c382.png" alt="picture">
  <div class="bg-style">.bg-style</div>
  <div class="app1">app1.css - imported in JS</div>
  <div class="app2">app2.css - imported in JS</div>
  <script src="app.ffec14fb.js" integrity="sha384-d3IyoBAyHtc5pCoPhJJaTiL1mjATSrumF8ih7vHr0yw8q6LtxFPGBDs7L8iojOKO" crossorigin="anonymous"></script>
</body>
</html>